apis/types/resources.go

```diff
@@ -36,7 +36,7 @@ type Resources struct {
 	//
 	BlkioDeviceWriteIOps []*ThrottleDevice `json:"BlkioDeviceWriteIOps"`
 
-	// Block IO weight (relative weight).
+	// Block IO weight (relative weight), need CFQ IO Scheduler enable.
 	// Maximum: 1000
 	// Minimum: 0
```

Comment:
between 10 and 1000
Minimum should be 0

Reply:
Yes, Could you help fix this, @sunyuan3